setup/prompt.ts

```diff
@@ -15,3 +15,19 @@ export const askSessionToken = async () => {
 
   return sessionToken;
 };
+
+export const askIOBackendBaseurl = async () => {
```

Comment:
there's no need prompt the user, reading from env suffices.

Reply:
Ok.
I'd leave it anyway because is works already with env variables, but it might be better DX for someone